app/src/main/java/com/example/life_is_well/BMICalc.kt

```diff
@@ -1,11 +1,83 @@
 package com.example.life_is_well
 
+import android.content.Context
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.widget.Button
+import android.widget.EditText
+import android.widget.TextView
+import android.widget.Toast
+import androidx.core.app.NotificationCompat.getColor
+import androidx.core.content.ContextCompat
 
 class BMICalc : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_bmicalc)
+
+        val weightText = findViewById<EditText>(R.id.etWeight)
+        val heightText = findViewById<EditText>(R.id.etHeight)
+        val calcButton = findViewById<Button>(R.id.btnCalculate)
```

Comment:
Nice! This is much better than importing entire xml files into the .kt files!

Reply:
@jacobburgo that is exactly my thought.  Much better data management.